modules/standard/Active/ActiveTest.enc

```diff
@@ -1,12 +1,13 @@
--- This file was automatically converted by encorec
-
 import Active.Active
```

Comment:
Shouldn't this test be moved to the test directory?
(Yes, you can argue that you didn't put it here, but as you are modifying the test, then now is the time!!)